Editor/Interaction/SnapZoneEditor.cs

```diff
@@ -66,7 +64,6 @@ public override void OnInspectorGUI()
 
             EditorGUILayout.PropertyField(interactionManager, Tooltips.InteractionManager);
             EditorGUILayout.PropertyField(interactionLayerMask, Tooltips.InteractionLayerMask);
-            EditorGUILayout.PropertyField(attachTransform, Tooltips.AttachTransform);
```

Comment:
Not nice

Reply:
This iteration does not support custom attach transforms. We can always enable it back later.

Reply:
A use case is disabling a native feature and forcing the snap zones not to be usable but with the snap zone creator.
Also, this implementation does not guarantee backward compatibility so old snap zones have to be rebuild.

Sorry but I don't think this is the way to go.

Reply:
Does the highlight object is disabled when the snap zone is not active? At the beginning of a course could seem like that but actually the highlight object does not yet exist, after an object was snapped the highlight is disabled but, what if in the same course the user is required to grab the object from the snap zone and place it somewhere else? The snap zone would be disabled but I am not sure if the highlight object would be shinning or not after the snap zone is disabled 🤔

Also, in case we go with this approach, there is no need to have a validation color and material here neither in the snap zone settings, we should deprecate them.